lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +20,15 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+    text = text.lower()
+    cleaned_text = ''
+    for i in text:
+        if i.isalpha():
+            cleaned_text += i
```

Comment:
concatenation of strings is a bad way, especially then you return list

Reply:
done